public/app/core/components/AppChrome/TopBar/TopNavBarMenuProgressBar.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+interface Props {
+  level: string;
+  totalAchievement: number;
+  currentAchievement: number;
+}
+
+export function TopNavBarMenuProgressBar({ level, totalAchievement, currentAchievement }: Props) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div>
+      <div className={styles.progressBarHeader}>
+        Level: <span>{level}</span>
+      </div>
+      <div className={styles.progressBar}>
+        <span
+          className={styles.progressBarFill}
+          style={{ width: `${(currentAchievement / totalAchievement) * 100}%` }}
+        ></span>
+      </div>
+      <span className={styles.progressBarCaption}>
+        {`${currentAchievement} of ${totalAchievement} achievements earned`}
```

Comment:
Feedback - we can remove this for time being and just have progress bar be split up into 5 level segments, filled up to current level + % of the users progress in that current level tier